testsuite/src/Test/Tasty/Clash.hs

```diff
@@ -27,12 +28,25 @@ import Test.Tasty.Program
 data BuildTarget
   = VHDL
   | SystemVerilog
-  | Verilog
-  deriving (Show, Eq, Ord)
+  | VerilogX Bool
```

Comment:
I think this should be an option in `TestOptions`.

Reply:
Made it a TestOption